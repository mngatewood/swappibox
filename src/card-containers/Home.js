import React from 'react';
import Stats from '../Utility/Stats'

const Home = () => {
  const displayHome = "Select a Category"
    return (
      <div>
        <h2>
          {displayHome}
        </h2>
      </div>
    )
}

export default Home;