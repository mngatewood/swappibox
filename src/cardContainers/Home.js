import React from 'react';

const Home = () => {
  const displayHome = "Select a Category";
  return (
    <div className="displayHome">
      <h2>
        {displayHome}
      </h2>
    </div>
  );
};

export default Home;