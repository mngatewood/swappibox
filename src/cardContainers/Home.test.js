import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Home', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Home />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match the snapshot', () => {
    const wrapper = shallow(<Home />);   
    expect(wrapper).toMatchSnapshot();
  });

});
