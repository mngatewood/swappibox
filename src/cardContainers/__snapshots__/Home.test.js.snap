// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h2>
        Select a Category
</h2>,
      "className": "displayHome",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Select a Category",
      },
      "ref": null,
      "rendered": "Select a Category",
      "type": "h2",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Select a Category
</h2>,
        "className": "displayHome",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Select a Category",
        },
        "ref": null,
        "rendered": "Select a Category",
        "type": "h2",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
